sympy/core/function.py

```diff
@@ -1339,7 +1339,7 @@ def __new__(cls, expr, *variables, **kwargs):
         if isinstance(expr, Derivative):
```

Comment:
Would add if evaluate

Reply:
![Screenshot from 2019-03-23 13-43-12](https://user-images.githubusercontent.com/44258119/54863696-d7486580-4d71-11e9-9230-b415777c3a63.png)
is it correct now?

Reply:
that works, can be one lined `if isinstance(expr, Derivative) and evaluate:`